app/connectors/CustomsDeclareExportsConnector.scala

```diff
@@ -84,7 +84,7 @@ class CustomsDeclareExportsConnector @Inject() (appConfig: AppConfig, httpClient
     val pagination = models.Page.bindable.unbind("page", page)
     val sort = DeclarationSort.bindable.unbind("sort", DeclarationSort(SortBy.UPDATED, SortDirection.DES))
 
-    httpClient.GET[Paginated[ExportsDeclaration]](url(s"${appConfig.declarationsPath}?status=DRAFT&$pagination&$sort"))
+    httpClient.GET[Paginated[ExportsDeclaration]](url(s"${appConfig.declarationsPath}?status=DRAFT&status=AMENDMENT_DRAFT&$pagination&$sort"))
```

Comment:
I've realised we need to flag this too, so when the flag is disabled we don't want to pull through any AMENDMENT_DRAFTS